Formula/python@3.11.5.rb

```diff
@@ -0,0 +1,527 @@
+class PythonAT311 < Formula
```

Comment:
Shouldn't it be `PythonAT3115` so we could have another formula for a later version in parallel?

Reply:
good catch!